app/Http/Controllers/UserController.php

```diff
@@ -161,6 +161,10 @@ public function update(Request $request, $id)
             $user->external_auth_id = $request->get('external_auth_id');
         }
 
+        if ($request->filled('email') !== $request->old('email')) {
+            $this->userRepo->downloadGravatarToUserAvatar($user);
```

Comment:
If enabled, This method call will always replace the user's existing profile image with the gravatar image. I'd assume the user wouldn't want their profile picture replaced with a gravatar image if they've previously customised it. I think we'd need to account for this.